src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -215,15 +212,40 @@ public static boolean isRadioactive(@Nullable ItemStack item) {
         return SlimefunItem.getByItem(item) instanceof Radioactive;
     }
 
+    private static final UUID RANDOM_UUID = UUID.fromString("92864445-51c5-4c3b-9039-517c9927d1b4");
+
     /**
-     * This method returns an {@link ItemStack} for the given texture.
-     * The result will be a Player Head with this texture.
-     *
-     * @param texture
-     *            The texture for this head (base64 or hash)
-     *
-     * @return An {@link ItemStack} with this Head texture
+     * This method returns a {@link PlayerProfile} for the given texture url.
+     * @param url The URL to the texture
+     * @return A {@link PlayerProfile} with the given texture
      */
+    private static PlayerProfile getProfile(@Nonnull String url) {
+        Validate.notNull(url, "The provided url is null");
+
+        PlayerProfile profile = Bukkit.createPlayerProfile(RANDOM_UUID); // Create a new player profile
+        PlayerTextures textures = profile.getTextures();
+        URL urlObject;
```

Comment:
URL is deprecated please use URI

Reply:
Are you asking for me to make it a URI and then convert it to URL? <PlayerTextures>.seSkin only accepts a URL parameter.